cpp/src/umap/knn_graph/algo.cuh

```diff
@@ -62,6 +62,7 @@ inline void launcher(const raft::handle_t& handle,
   ptrs[0]  = inputsA.X;
   sizes[0] = inputsA.n;
 
+  std::vector<int64_t>* translations = nullptr;
```

Comment:
Any reason to introduce a temporary for this?

Reply:
The function template won't be instantiated while providing a `nullptr` directly unless a cast is used it seems. I just switched it for a cast.